src/OSPSuite.Core/Domain/Services/ConcurrencyManager.cs

```diff
@@ -82,26 +84,25 @@ public class ConcurrencyManager : IConcurrencyManager
                var result = await returnWithExceptionHandling(
                   coreIndex,
                   cancellationToken,
-                  action, 
+                  action,
                   datum,
                   id
                );
                results.TryAdd(datum, result);
             }
-         }).ToList();
+         });
 
          await Task.WhenAll(tasks);
          //all tasks are completed. Can return results
 
-         var tt = results.Values;
```

Comment:
not required

Reply:
Looks like something that was left behind when debugging.